### PR TITLE
ansible-test - Hide ansible._vendor in import test

### LINKED ISSUE #76473: Ansible-test import fails - one or more Python packages bundled by this ansible-core
### Summary

Ansible-test import fails in our CI with one or more Python packages bundled by this ansible-core 

### Issue Type

Bug Report

### Component Name

ansible-test

### Ansible Version

```console
$ ansible --version
ansible [core 2.12.0]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.9/site-packages/ansible
  ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.9.8 (main, Nov  8 2021, 00:00:00) [GCC 11.2.1 20211019 (Red Hat 11.2.1-6)]
  jinja version = 2.11.3
  libyaml = True
```

### Configuration

```console
$ ansible-config dump --only-changed
```

### OS / Environment

container centos:stream9

### Steps to Reproduce

```yaml (paste below)

```

### Expected Results

Pass the test

### Actual Results

```console
https://github.com/oVirt/ovirt-ansible-collection/runs/4391847035?check_suite_focus=true


WARNING: The directory '/github/home/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you should use sudo's -H flag.
Collecting cryptography<3.4
  Downloading cryptography-3.3.2-cp36-abi3-manylinux2010_x86_64.whl (2.6 MB)
Collecting six>=1.4.1
  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting cffi>=1.12
  Downloading cffi-1.15.0-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (444 kB)
Collecting pycparser
  Downloading pycparser-2.21-py2.py3-none-any.whl (118 kB)
Installing collected packages: pycparser, six, cffi, cryptography
Successfully installed cffi-1.15.0 cryptography-3.3.2 pycparser-2.21 six-1.16.0
WARNING: The directory '/github/home/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you should use sudo's -H flag.
Collecting jinja2
  Downloading Jinja2-3.0.3-py3-none-any.whl (133 kB)
Collecting PyYAML
  Downloading PyYAML-6.0-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (661 kB)
Collecting packaging
  Downloading packaging-21.3-py3-none-any.whl (40 kB)
Collecting resolvelib<0.6.0,>=0.5.3
  Downloading resolvelib-0.5.4-py2.py3-none-any.whl (12 kB)
Collecting MarkupSafe>=2.0
  Downloading MarkupSafe-2.0.1-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (30 kB)
Collecting pyparsing!=3.0.5,>=2.0.2
  Downloading pyparsing-3.0.6-py3-none-any.whl (97 kB)
Installing collected packages: pyparsing, MarkupSafe, resolvelib, PyYAML, packaging, jinja2
Successfully installed MarkupSafe-2.0.1 PyYAML-6.0 jinja2-3.0.3 packaging-21.3 pyparsing-3.0.6 resolvelib-0.5.4
ERROR: Command "importer.py" returned exit status 10.
>>> Standard Error
/tmp/ansible-test-ibrisv3_/ansible/_vendor/__init__.py:42: UserWarning: One or more Python packages bundled by this ansible-core distribution were already loaded (packaging). This may result in undefined behavior.
  warnings.warn('One or more Python packages bundled by this ansible-core distribution were already '
/tmp/ansible-test-ibrisv3_/ansible/_vendor/__init__.py:42: UserWarning: One or more Python packages bundled by this ansible-core distribution were already loaded (packaging). This may result in undefined behavior.
  warnings.warn('One or more Python packages bundled by this ansible-core distribution were already '
/tmp/ansible-test-ibrisv3_/ansible/_vendor/__init__.py:42: UserWarning: One or more Python packages bundled by this ansible-core distribution were already loaded (packaging). This may result in undefined behavior.
  warnings.warn('One or more Python packages bundled by this ansible-core distribution were already '
/tmp/ansible-test-ibrisv3_/ansible/_vendor/__init__.py:42: UserWarning: One or more Python packages bundled by this ansible-core distribution were already loaded (packaging). This may result in undefined behavior.
  warnings.warn('One or more Python packages bundled by this ansible-core distribution were already '
/tmp/ansible-test-ibrisv3_/ansible/_vendor/__init__.py:42: UserWarning: One or more Python packages bundled by this ansible-core distribution were already loaded (packaging). This may result in undefined behavior.
  warnings.warn('One or more Python packages bundled by this ansible-core distribution were already '
>>> Standard Output
plugins/inventory/ovirt.py:85:0: traceback: UserWarning: One or more Python packages bundled by this ansible-core distribution were already loaded (packaging). This may result in undefined behavior.
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by mattclay**
##### SUMMARY

ansible-test - Hide ansible._vendor in import test

Fixes https://github.com/ansible/ansible/issues/76473

##### ISSUE TYPE

Bugfix Pull Request

##### COMPONENT NAME

ansible-test

### CODE REVIEW

File: test/integration/targets/ansible-test/ansible_collections/ns/col/plugins/lookup/vendor1.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+name: vendor2
```
Comment (nitzmahone) [inline test/integration/targets/ansible-test/ansible_collections/ns/col/plugins/lookup/vendor1.py]: vendor1?

**Review by nitzmahone — CHANGES_REQUESTED** [review]
one minor copy/paste thing, otherwise LGTM

File: test/integration/targets/ansible-test/ansible_collections/ns/col/plugins/lookup/vendor1.py (hunk 1 above)
Comment (mattclay) [inline test/integration/targets/ansible-test/ansible_collections/ns/col/plugins/lookup/vendor1.py]:
```suggestion
name: vendor1
```

Comment (mattclay) [reply test/integration/targets/ansible-test/ansible_collections/ns/col/plugins/lookup/vendor1.py]: Fixed.

**Review by nitzmahone — APPROVED** [review]

**Pull request merged**